### DISCUSSION TITLE
Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by HarryOMalley** [sindresorhus/ky#426] [Q&A] [answered]
When I try to add Ky to my project, I get the following error when attempting to run tests with Jest:

`
Details:

    /Users/harryomalley/desert-island-records/client/node_modules/ky/distribution/index.js:2
    import { Ky } from './core/Ky.js';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      14 | import { RateLimiter } from './rate-limiter';
      15 | import * as mb from './musicbrainz.types';
    > 16 | const ky = require('ky');
         |            ^
      17 | // import ky, { Options } from 'ky';
      18 | import * as tough from 'tough-cookie';
      19 |

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1728:14)
      at Object.<anonymous> (src/api/musicbrainz/musicbrainz-api.ts:16:12)
`

At first I thought I had a configuration wrong with my project, before I realised that it is complaining about the import statement in Ky itself. Does anyone know how I can fix this?

**Comment by sholladay** [comment, marked as answer]
Jest dose some weird transform stuff to your dependencies, which breaks ES modules.

They have experimental support for turning that stuff off, finally.

https://jestjs.io/docs/ecmascript-modules

**Reply by HarryOMalley** [reply]
Thanks @sholladay !
